.github/workflows/migrations-check.yml

```diff
@@ -15,8 +15,12 @@ jobs:
       matrix:
         os: [ ubuntu-20.04 ]
         python-version: [ 3.8 ]
+        django-version: ["pinned", "4.2"]
         mongo-version: ["4"]
         mysql-version: ["5.7", "8"]
+        exclude:
```

Comment:
A comment about why we're excluding this would also be helpful

Reply:
Django 4.2 has dropped support for MySQL<8 so we've excluded the combination MySQL57+Django42